src/web/handlers/TapGestureHandler.ts

```diff
@@ -61,8 +61,8 @@ export default class TapGestureHandler extends GestureHandler {
       this.maxDeltaY = this.config.maxDeltaY;
     }
 
-    if (this.config.maxDistSq !== undefined) {
-      this.maxDistSq = this.config.maxDistSq;
+    if (this.config.maxDist !== undefined) {
+      this.maxDist = this.config.maxDist;
     }
```

Comment:
```suggestion
    if (this.config.maxDistSq !== undefined) {
      this.maxDistSq = this.config.maxDist * this.config.maxDist;
    }
```

Right now, in the `shouldFail` function, you're comparing squared distance to non-squared one